go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/demisto/thirdPartyLicenseCollector
```

Comment:
is it valid that the github repo is not actually placed in `github.com/demisto` ?

Reply:
Not sure I understand the question. The go.mod file is not part of my commit. My changes are 1660729 & 7f61bb0.